examples/models/toolbars.py

```diff
@@ -1,3 +1,12 @@
+''' This example shows multiple ways to place the toolbar and the x and y axes 
```

Comment:
trailing space crept in, will be ready to merge after this is fixed. 
```suggestion
''' This example shows multiple ways to place the toolbar and the x and y axes
```
FYI it is advied to run codebase checks before pushing, all these kinds of issues can be caught locally

Reply:
Will keep in mind, thank you for the feedback!

Reply:
Made the changes @bryevdv